app/models/support_interface/provider_user_form.rb

```diff
@@ -6,7 +6,7 @@ class ProviderUserForm
     attr_accessor :first_name, :last_name, :provider_ids, :provider_user
     attr_reader :email_address
 
-    validates :email_address, presence: true
+    validates :email_address, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
```

Comment:
We have a `EmailAddressValidator` that you can use: 

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/a6e4f86e283a349a13fb667ce406edf1034343e8/app/models/application_reference.rb#L6